app/models/conus_linehaul.rb

```diff
@@ -0,0 +1,2 @@
+class ConusLinehaul < Linehaul
+end
```

Comment:
I don't know how common this pattern is, but I've seen others do something like the following when there's a base parent class and a series of empty child classes:

```rb
# in /app/models/linehaul.rb...

class Linehaul < ApplicationRecord
  # your base class' stuff here...
end

class ConusLinehaul < Linehaul; end
class IntraAlaskaLinehaul < Linehaul; end
# etc. etc. etc.
```

This pattern removes a bunch of nearly-empty files from `app/models` which can be both a good and a bad thing. I'm not entirely sure how I feel about either approach.

Reply:
I understand the appeal, but for now, I would rather leave them as separate files.